Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/Batching/Batching.xaml.cs

```diff
@@ -0,0 +1,240 @@
+            var session = new LearningModelSession(model, device, options);
+            return session;
+        }
+
+        private static LearningModel CreateLearningModel(string modelPath)
+        {
+            var uri = new Uri(modelPath);
+            var file = StorageFile.GetFileFromApplicationUriAsync(uri).GetAwaiter().GetResult();
+            return LearningModel.LoadFromStorageFileAsync(file).GetAwaiter().GetResult();
+        }
+
+        async private System.Threading.Tasks.Task Classify(List<VideoFrame> inputImages)
+        {
+            float totalEvalDurations = 0;
+            for (int i = 0; i < 100; i++)
+            {
+                if (navigatingAwayFromPage)
+                    break;
+                UpdateEvalProgressUI(i);
+                float evalDuration = await System.Threading.Tasks.Task.Run(() => Evaluate(nonBatchingSession_, inputImages));
+                totalEvalDurations += evalDuration;
+            }
+            avgNonBatchedDuration = totalEvalDurations / 100;
+        }
+
+        private static float Evaluate(LearningModelSession session, List<VideoFrame> input)
+        {
+            string inputName = session.Model.InputFeatures[0].Name;
+            float totalDuration = 0;
+            for (int j = 0; j < input.Count; j++)
+            {
+                if (navigatingAwayFromPage)
+                    break;
+                var binding = new LearningModelBinding(session);
+                var start = HighResolutionClock.UtcNow();
+                binding.Bind(inputName, input[j]);
+                session.Evaluate(binding, "");
+                var stop = HighResolutionClock.UtcNow();
+                var duration = HighResolutionClock.DurationInMs(start, stop);
+                totalDuration += duration;
+            }
+            return totalDuration;
+        }
+
+        async private System.Threading.Tasks.Task ClassifyBatched(List<VideoFrame> inputImages, int batchSize)
+        {
+            float totalEvalDurations = 0;
+            for (int i = 0; i < 100; i++)
+            {
+                if (navigatingAwayFromPage)
+                    break;
+                UpdateEvalProgressUI(i);
+                float evalDuration = await System.Threading.Tasks.Task.Run(() => EvaluateBatched(BatchingSession_, inputImages, batchSize));
+                totalEvalDurations += evalDuration;
+            }
+            avgBatchDuration = totalEvalDurations / 100;
+        }
+
+        private static float EvaluateBatched(LearningModelSession session, List<VideoFrame> input, int batchSize)
+        {
+            // Evaluate
+            int numBatches = input.Count / batchSize;
+            string inputName = session.Model.InputFeatures[0].Name;
+            float totalDuration = 0;
+            for (int i = 0; i < numBatches; i++)
+            {
+                if (navigatingAwayFromPage)
+                    break;
+                var binding = new LearningModelBinding(session);
```

Comment:
Reuse bindings instead of creating new ones every time

Reply:
Done.